src/Traits/QueryBuilder.php

```diff
@@ -47,7 +47,7 @@ private function buildQueryParameters(array $parameters = []): string
 
         foreach ($parameters as $column => $query) {
             if ($column === 'load_relations' && is_array($query)) {
-                $queryParameters[] = $column . $this->_getOperator($this->operator_equal) . sprintf('["%s"]', implode(',', $query));
```

Comment:
At this point, it might be better to just `json_encode` the `$query` no?

```php
$column . $this->_getOperator($this->operator_equal) . json_encode($query);
```
That should give the same result.

Reply:
@BOAWEB could you have a look at this please?